src/www/messages/en.json

```diff
@@ -19,6 +19,7 @@
   "error-server-incompatible": "Sorry, this access key is not compatible with this version of Outline.",
   "error-timeout": "Something seems to be taking longer than expected. Quitting and restarting may help. If this happens again, please submit feedback.",
   "error-unexpected": "Sorry, an unexpected error occurred. Quitting and restarting may help. If this happens again, please submit feedback.",
+  "error-unsupported-cipher": "Sorry, the server you are trying to connect to uses an unsupported cipher. Non-AEAD ciphers are not supported.",
```

Comment:
We need a more actionable error. Maybe something like "The server you added uses cipher {cipher}, but Outline only supports AEAD ciphers. Please contact the admin of the server."

Reply:
Done